### ISSUE TITLE
Some NVDA Problems i have noticed.

### ISSUE THREAD

**Issue by aefren** [nvaccess/nvda#8182]
Here some of my problems with NVDA.

I press the start menu to search something.
I write "notepad" for example and press enter.
then NVDA stop reading explorer's items until I restart NVDA.
However Explorer's items can be read by pressing the eight numpad key.

NVDA is going slower. Something when I have lot of application running NVDA becomes laggy. stop reading program title when I press alt plus tab or reads the windows explorer content after some seconds and this problem cannot be solved starting NVDA nor pressing the eight numpad key.

This problem only occur only when I’m using NVDA. The other screen reader "jaws" can work even with lot of application running at same time. so I don't think it can be a lack of pc resources.
I see this problem since NVDA 2017 maybe or 2016, when I was running a 32 bits windows 10 pc. Now I have windows 10 64 bits, 8gb ram, i5 processor at 3.20hz

I also noticed that NVDA is now less accessible to p2p programs like emule, or Tixati.

I always run an installed NVDA.
current NVDA version is 2018.

**Comment by Neurrone** [comment]
Sounds very similar to #8145. 

Are you by any chance using the power saver power plan? If so, try changing it to balanced to see if that helps.

**Comment by Brian1Gaff** [comment]
I had to make sure idle was 50 percent to really make a difference to this 
sort of issue, of course I'm using a desktop so its no problem to use more 
power.

I'd like to know hy Windows seems unable to notice that nvda is making 
demands on a core that max it out though. Would seem to me this is something 
they need to address, but also if as has been said, Jaws avoids it, it might 
be interesting to know what they are doing.

It happens to some extent on Windows 7 as this is what I use. However the 
problems with software not running the same could be almost anything.

Brian

**Comment by Neurrone** [comment]
I suspect its because of the power saver settings. It doesn't push the CPU to higher performance states unless its being taxed enough - the fact that NVDA doesn't tax the CPU enough to push it to higher performance is actually a good thing.

Regardless though, the responsiveness issues in windows explorer still occur though not as severely once power plans are no longer an issue. Still hope to see it fixed.

**Comment by aefren** [comment]
I use NVDA in a desktop pc. my energy plan is in hight performance. Is what are you talking  when says "power saver settings"right? my pc is in spanish so i thing "power saver settings" is where you configures how minuts before to turn off the screen, the disk etc.

NVDA is the only laggy application, all others run finely. when I check the taskbar performance tab i see the CPU running slightly less of 50% or slightly above 50% when the problem start
talking with some friend they have tell me about similars problems.

**Comment by Brian1Gaff** [comment]
No there is another setting that how much the processor is throttled back. I know where it is in windows 7, but not 10,so will bow to others explanations on where to find it,.

Brian

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
This message sent from a Windows XP machine!
  ----- Original Message -----

**Comment by Neurrone** [comment]
@aefren it shouldn't be lagging even with high performance. What processor is the computer using?

**Comment by netblue44** [comment]
I have also noted this problem as well.

I am using Windows 10 PRO 64 bits with latest snaps of NVDA.

**Comment by aefren** [comment]
I don't think is my processor. There is more people having the same problem. special with the laggy windows explorer.

There is a throtle column in taskbar/details where you can see wich application is being throtled. is not the case of NVDA.
This column must be enabled clicking in the column header in the detail tab.

**Comment by Adriani90** [comment]
Closing.
@aefren please create individual issues for each problem. This makes it easier to follow the discussion. Thanks.
